project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module

```diff
@@ -520,6 +528,45 @@ function c4m_message_format_group_node_type(Message $message) {
 }
 
 /**
+ * Returns message to admin token.
+ *
+ * @param Message $message
+ *   The Message entity.
+ *
+ * @return string
+ *   Return message to admin, which is entered by group requester.
+ */
+function c4m_message_format_message_to_admins(Message $message) {
```

Comment:
@anvmn 
I think it will be more clear if we'll add the word _group_ to the name of the function and also to the docs
> Returns message to admin token

seems to general ...